src/index.html

```diff
@@ -6,8 +6,21 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta name="description" content="TV Show Web App">
   <title>TV Show Web App</title>
+  <link rel="stylesheet" href="./styles/styles.css">
```

Comment:
This line is unnecessary since the config of the webpack is based on the Javascript, index.js. I'll remove it in my next update

Reply:
well noticed